layerfetcher/wire_types.go

```diff
@@ -2,7 +2,17 @@ package layerfetcher
 
 import "github.com/spacemeshos/go-spacemesh/common/types"
 
-// layerBlocks is the response for layer hash
+// layerHash is the response for a given layer
+type layerHash struct {
+	// ProcessedLayer is the latest processed layer from peer
+	ProcessedLayer types.LayerID
+	// SimpleHash is the hash of contextually valid blocks (sorted by block ID) in the given layer
+	SimpleHash types.Hash32
```

Comment:
i think just Hash is easier to understand compared with SimpleHash

Reply:
done